GameServerCore/IPlayerManager.cs

```diff
@@ -13,5 +13,6 @@ public interface IPlayerManager
         ClientInfo GetClientInfoByChampion(IChampion champ);
         ClientInfo GetPeerInfo(long playerId);
         List<Tuple<uint, ClientInfo>> GetPlayers(bool includeBots = false);
+        Tuple<uint, ClientInfo> GetClientInfoWithPlayerId(int playerId);
```

Comment:
Remove this function and simply pass in the ClientInfo to the packet notifier functions which used this.
```suggestion
```